components/eam/src/chemistry/modal_aero/modal_aero_convproc.F90

```diff
@@ -1920,6 +1794,9 @@ subroutine compute_activation_tend(                          &
 
    logical      :: do_act_this_lev      ! flag for doing activation at current level
    integer      :: kp1                  ! kk + 1
+   real(r8), parameter :: clw_cut = 1.0e-6      ! cutoff value of cloud water for doing updraft [kg/kg]
```

Comment:
Needs `_r8`?

Reply:
Same here. This number is also used without `_r8` in the original code.

Reply:
as commented by Balwinder, I will keep the numbers as it as the original code